framework/Source/iOS/GPUImageMovieWriter.h

```diff
@@ -34,6 +34,7 @@ extern NSString *const kGPUImageColorSwizzlingFragmentShaderString;
 @property(readwrite, nonatomic) BOOL hasAudioTrack;
 @property(readwrite, nonatomic) BOOL shouldPassthroughAudio;
 @property(readwrite, nonatomic) BOOL shouldInvalidateAudioSampleWhenDone;
+@property(nonatomic, copy) void(^didProcessFrameAtTimeBlock)(CMTime);
```

Comment:
Do you need to pass back a locked frames-buffer or anything like that?

Reply:
I don't think so - all I need is the time of the frame the movie writer has finished processing